Cargo.toml

```diff
@@ -53,6 +53,7 @@ unstable = []
 ipv6 = []
 cuda = []
 erasure = []
+gossip_choose_weighted_peer = []
```

Comment:
I don't think this needs to be guarded with a feature flag. It's nice to have both options available, but I'm hoping a "best" option will emerge and we'll just always use that one.

Reply:
ah ok, I was wondering if it was necessary for a safe rollout procedure. I can remove this.